code/game/objects/structures/vehicles/wheelchair.dm

```diff
@@ -19,6 +19,16 @@
 	. = ..()
 	wheel_overlay = image("icons/obj/objects.dmi", "[icon_state]_overlay", MOB_LAYER + 0.1)
 
+/obj/structure/bed/chair/vehicle/wheelchair/attackby(obj/item/weapon/W, mob/user)
+	if(occupant)
+		return
+	if(istype(W, /obj/item/weapon/gun_barrel))
+		to_chat(user, "You place \the [W] on \the [src].")
+		var/obj/structure/bed/chair/vehicle/wheelchair/wheelchair_assembly/I = new (get_turf(src.loc))
+		I.dir = dir
+		qdel(src)
+		qdel(W)
```

Comment:
Couldn't this be fixed for every such case by just putting an
```
if(ismob(loc))
     var/mob/M = loc
     M.drop_item(src)
```

Or something similar into the Destroy() of `/obj/item`? It'd make it so that the act of qdel()ing an item causes the item to automatically be dropped if it's being held.